lesson08/assignment/test-project/src/main.js

```diff
@@ -0,0 +1,22 @@
+import Vue from 'vue'
+import App from './App.vue'
+import router from './router'
+import { BootstrapVue, IconsPlugin } from 'bootstrap-vue'
+
```

Comment:
Add an import for bootstrap here.  This is for bootstrap 5, not BootstrapVue.

import 'bootstrap'